QUARKUS-280.md

```diff
@@ -0,0 +1,31 @@
+## QUARKUS-280 - Tech Preview support for Message tracing and Open Telemetry
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-280
+
+OpenTelemetry extension for Quarkus is a preferred future solution for the application's telemetry. The current OpenTracing library for telemetry from Smallrye is deprecated and will not be supported in the future.
+Quarkus brings three extenions related to OpenTelemetry:
+ - quarkus-opentelemetry
+ - quarkus-opentelemetry-exporter-jaeger
+ - quarkus-opentelemetry-exporter-otlp
+
```

Comment:
Add this text `This feature request is supposed to focus primarily on quarkus-opentelemetry and quarkus-opentelemetry-exporter-jaeger, according to PM they are the demanded by other products.` on new line.